src/components/Homepage/events/DesktopMeetups.js

```diff
@@ -18,19 +19,15 @@ const DesktopMeetups = ({ events }) => (
     <Padding top={42}>
       <ul>
         {getHomepageMeetups(events).map(event => (
-          <CustomisedBulletpoint
-            fullWidth
-            fullWidthBorder
-            symmetrical
-            key={`${event.id}`}
-          >
+          <li key={`${event.id}`}>
```

Comment:
since this is also used on the homepage, would it make sense to create a component for this?

Reply:
If it's not 100% the same, forget what I just said